source/_docs/z-wave.markdown

```diff
@@ -18,8 +18,6 @@ Before configuring the Z-Wave setup, please take a moment and read [this article
 
 ### {% linkable_title Installation %}
 
-As of version 0.45, Home Assistant automatically installs python-openzwave from PyPI as needed.
```

Comment:
After removing the version reference, I was originally going to move this below the installation steps shown. As it's written right now, the section starts off sounding like home assistant will automatically install what it needs, but then we give further instructions below. (Bad for skimming)

However, I'm thinking the line is unnecessary entirely. Home Assistant always auto-installs requirements. It's not really necessary to bring up here. We still have info about first run install time below anyway.

Reply:
Agreed.